### PR TITLE
added login.html file and fixed newline 

### PR DESCRIPTION
**Pull request by OdedHellman**

### CODE REVIEW

File: superx/routes/home.py [hunk 1]
```diff
@@ -2,3 +2,7 @@
  
 def home():
     return render_template('home.html')
+
+def login():
+	return render_template('login.html')
+
```
Comment (simzacks) [inline superx/routes/home.py]: The last line here isn't blank, it's a tab.

File: superx/routing.py [hunk 2]
```diff
@@ -4,3 +4,8 @@
 @app.route('/')
 def index_route():
     return home.home()
+
+@app.route('/login')
+def login():
+	return home.login()
+
```
Comment (simzacks) [inline superx/routing.py]: The last line here isn't blank, it's a tab.

File: superx/routing.py [hunk 3]
```diff
@@ -4,3 +4,8 @@
 @app.route('/')
 def index_route():
     return home.home()
+
+@app.route('/login')
+def login():
```
Comment (simzacks) [inline superx/routing.py:9]: You might want to have a consistent naming scheme.
index is called "index_route", I would suggest either changing that to index or changing login to "login_route"

**Review by simzacks — CHANGES_REQUESTED** [review]

File: superx/routing.py (hunk 3 above)
Comment (OdedHellman) [reply superx/routing.py:9]: Changed "index_route" into "index".

**Comment by OdedHellman** [comment]
## sign in:
image
## sign up:
image
## Landing-Page:
image

**Review by simzacks — APPROVED** [review]

**Pull request merged**